rancher2/schema_cluster_oke_config.go

```diff
@@ -201,6 +203,18 @@ func clusterOKEConfigFields() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "Additional CIDR from which to allow ingress to worker nodes",
 		},
+		"pod_cidr": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Default:     "10.242.0.0/16",
+			Description: "Optional specify the pod CIDR",
+		},
+		"service_cidr": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Default:     "10.92.0.0/16",
+			Description: "Optional specify the service CIDR",
+		},
```

Comment:
Both arguments are set as `optional` but setting a default value, both values are required?? I mean, can't be these parameters let empty `""` at some oke deployment scenario??  Just checking if would be better to set both arguments also as computed, instead of setting default values

Reply:
TF fields can have both properties, `Optional` (user configurable) and `Computed` (not user configurable but getting data from provider source). 

Anyway, the question is not around my preference to be Computed fields or not, moreover it would be howto facilitate use case. If the fields are optional, i assume that the fields can be empty. So if default value has been defined, for a use case where the user wants to let field empty it should defined `pod_cidr=""`. Other option to set the fields as Computed, may be not set default value or set it as `Default: ""` . I'm just asking as i don't have any experience using oke_config. If this config is fine for you, LGTM.. :)

Reply:
What this PR is trying to achieve is to add the ability to define those variables for the user who might want alternate CIDR's for their cluster pods and services.  If the user isn't worried about it, they would not even define pod_cidr or service_cidr in the oke_config which would result in them getting the OKE defaults for cluster creation.  Here is a snippet of the cluster being created with the option in question.  We want the user to not even define those if they don't need to and have the cluster create work, which is why it's Optional.

<img width="511" alt="Screen Shot 2021-05-12 at 11 50 27 AM" src="https://user-images.githubusercontent.com/30269407/118030260-e7d54480-b319-11eb-87a5-aabcdbc0967b.png">

There is a similar example for a similar variable in the 'schema_cluster_aks_config'.

Reply:
@rawmind0, I  get your point now, I removed the Default settings for these parameters.